### PR TITLE
SPOI-9079 Added sample application for transform operator

### PR DESCRIPTION
**Pull request by chaithu14**

### CODE REVIEW

**Comment by chaithu14** [comment]
@chinmaykolhatkar Please review and merge.

File: tutorials/README.md [hunk 1]
```diff
@@ -18,4 +18,4 @@
 | parser            | Examples showing how to use different parsers and formatters |
 | partition         | Shows use of custom partitioning with a stream codec.
 | topnwords         | The Top-N words example. |
-
```
Comment (chinmaykolhatkar) [inline tutorials/README.md]: Why this removal of line?

File: tutorials/transform/src/main/java/com/example/transform/CustomerEvent.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
+package com.example.transform;
+
+import java.util.Date;
+
+public class CustomerEvent
+{
+  public int customerId;
```
Comment (chinmaykolhatkar) [inline tutorials/transform/src/main/java/com/example/transform/CustomerEvent.java]: All the fields in tuple POJO should be private.

File: tutorials/transform/src/main/java/com/example/transform/CustomerInfo.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
+package com.example.transform;
+
+public class CustomerInfo
+{
+  public int customerId;
```
Comment (chinmaykolhatkar) [inline tutorials/transform/src/main/java/com/example/transform/CustomerInfo.java]: All the fields should be private.

File: tutorials/transform/src/main/java/com/example/transform/Application.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    expMap.put("address", "{$.address}.toLowerCase()");
+    transform.setExpressionMap(expMap);
+    ConsoleOutputOperator output = dag.addOperator("Output", new ConsoleOutputOperator());
+
+    dag.addStream("InputToTransform", input.output, transform.input);
+    dag.addStream("TransformToOutput", transform.output, output.input);
+
+    dag.setInputPortAttribute(transform.input, Context.PortContext.TUPLE_CLASS, CustomerEvent.class);
+    dag.setOutputPortAttribute(transform.output, Context.PortContext.TUPLE_CLASS, CustomerInfo.class);
+    dag.setAttribute(transform, Context.OperatorContext.PARTITIONER, new StatelessPartitioner<TransformOperator>(2));
```
Comment (chinmaykolhatkar) [inline tutorials/transform/src/main/java/com/example/transform/Application.java]: Maybe also have an application showing Dynamic partitioning. Please add steps in document to show how to drive dynamic partitioning. You can use Throughput based partitioner for it.

File: tutorials/README.md [hunk 5]
```diff
@@ -18,4 +18,4 @@
...
+| transform         | Shows how to use the transform operator. |
```
Comment (chinmaykolhatkar) [inline tutorials/README.md]: "Shows how to use the transform operator" -> "Example for Transform Operator"

**Review by chinmaykolhatkar — CHANGES_REQUESTED** [review]
Please take care of comments as mentioned.

**Comment by chinmaykolhatkar** [comment]
Also, did you run the applications in cluster mode?

File: tutorials/README.md (hunk 5 above)
Comment (chaithu14) [inline tutorials/README.md]: Done.

File: tutorials/transform/src/main/java/com/example/transform/CustomerEvent.java (hunk 2 above)
Comment (chaithu14) [inline tutorials/transform/src/main/java/com/example/transform/CustomerEvent.java]: Done.

**Comment by chaithu14** [comment]
Incorporated the review comments.

**Review by chinmaykolhatkar — APPROVED** [review]

**Pull request merged**